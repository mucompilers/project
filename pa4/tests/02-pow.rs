fn pow(x : i32, mut n : i32) -> i32 {
      let mut z = 1;
      while (n > 0) {
            z = z * x;
            n -= 1;
      };
      return z;
}

fn main() {
      println!("{}", pow(2, 10));
      println!("{}", pow(3, 4));
      println!("{}", pow(20, 2));
}
