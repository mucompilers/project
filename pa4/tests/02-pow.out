1024
81
400
